src/Api/Model/Shared/UserModel.php

```diff
@@ -14,7 +14,6 @@
 
 class UserModel extends MapperModel
 {
-    const COMMUNICATE_VIA_SMS = "sms";
     const COMMUNICATE_VIA_EMAIL = "email";
     const COMMUNICATE_VIA_BOTH = "both";
```

Comment:
This should be gone too.

```suggestion
```

Reply:
done in [5b1e679](https://github.com/sillsdev/web-languageforge/pull/1611/commits/5b1e679ade7f12fa97f2e8df43021736665bebbe)